pom.xml

```diff
@@ -263,7 +263,7 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>matrix-project</artifactId>
-      <version>1.4.1</version>
+      <version>1.4</version>
```

Comment:
Would it be better to drop this change from this commit and instead merge https://github.com/jenkinsci/git-plugin/pull/322 ?  That would remove one more pull request from the backlog and make this change explicit

Is there a matching change which will allow the JUnit plugin to work with 1.580 (see that preceding pull request for Kohsuke's reference to the JUnit special needed)